removal_from_ansible.rst

```diff
@@ -22,6 +22,11 @@ The general process of removing a collection follows these steps:
 #. Removing the collection.
 #. When appropriate, re-adding the collection.
 
+Where the community has `write` access to the collection repository, follow these steps:
```

Comment:
How about adding a new header for this section, so it can be referenced explicitly? And then reference it further below when it makes sense, like in the section on unmaintained collections. I don't think it makes sense to refer it from every removal process.